Steeltoe.ruleset

```diff
@@ -1,355 +1,63 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There are quite a lot of changes in this file that seem like they might be removing some suppressions, is all of this intentional?

Reply:
No the ruleset editor did something wierd - reverting ...

Reply:
Sorry I reverted but the diff in this looks weird - I reset it to what it was before.